crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1093,7 +1093,9 @@ where
         // Now broadcast the transaction to all authorities.
         let threshold = self.committee.quorum_threshold();
         let validity = self.committee.validity_threshold();
+        let tx_digest = transaction.digest();
         debug!(
+            digest = ?tx_digest,
```

Comment:
I wonder if we should actually always log it as tx_digest, since digest could mean many things